com.springrts.SpringLobby.yaml

```diff
@@ -0,0 +1,99 @@
+app-id: com.springrts.SpringLobby
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: springlobby
+rename-icon: springlobby
+rename-desktop-file: springlobby.desktop
+rename-appdata-file: springlobby.appdata.xml
+finish-args:
+- --share=network
+- --socket=x11
+- --share=ipc
+- --device=dri
+- --socket=pulseaudio
+- --persist=.spring
+- --persist=.springlobby
+
+modules:
+- name: glu
+  sources:
+    - type: archive
+      url: http://archive.ubuntu.com/ubuntu/pool/main/libg/libglu/libglu_9.0.1.orig.tar.gz
+      sha256: "f6f484cfcd51e489afe88031afdea1e173aa652697e4c19ddbcb8260579a10f7"
+  cleanup: 
+    - /include
+    - /lib/*.a
+    - /lib/*.la
+    - /lib/pkgconfig
+    - /man
+    - /share/aclocal
+    - /share/doc
+    - /share/gtk-doc
+    - /share/man
+    - /share/pkgconfig
+
+- name: alure
+  buildsystem: cmake-ninja
+  sources:
+  - type: archive
+    url: https://kcat.strangesoft.net/alure-releases/alure-1.2.tar.bz2
+    sha256: "465e6adae68927be3a023903764662d64404e40c4c152d160e3a8838b1d70f71"
+
+- name: wxwidgets
+  rm-configure: true
+  config-opts:
+    - --with-gtk=3
+    - --with-opengl
+    - --with-libjpeg
+    - --disable-shared
+    - --enable-monolithic--with-libtiff
+    - --with-libpng 
+    - --with-zlib
+    - --disable-sdltest
+    - --enable-unicode
+    - --enable-display
+    - --enable-propgrid
+    - --disable-webkit
+    - --disable-webview
+    - --disable-webviewwebkit
+    - --with-libiconv=/usr
+    - --without-subdirs
+    - --without-gtkprint
+    - --disable-gtktest
+  cleanup:
+    - /bin
+    - /lib/wx
+    - /share/bakefile
+  sources:
+    - type: script
+      dest-filename: autogen.sh
+      commands:
+        - cp -p /usr/share/automake-*/config.{sub,guess} .
+        - autoconf -f -B build/autoconf_prepend-include
+    - type: archive
+      url: https://github.com/wxWidgets/wxWidgets/releases/download/v3.0.4/wxWidgets-3.0.4.tar.bz2
+      sha256: 96157f988d261b7368e5340afa1a0cad943768f35929c22841f62c25b17bf7f0
+
+- name: boost
+  buildsystem: simple
+  build-commands:
+  - ./bootstrap.sh --prefix=/app --with-libraries=system,thread
+  - ./b2 install -j $FLATPAK_BUILDER_N_JOBS
+  sources:
+  - sha256: 8f32d4617390d1c2d16f26a27ab60d97807b35440d45891fa340fc2648b04406
+    type: archive
+    url: https://dl.bintray.com/boostorg/release/1.69.0/source/boost_1_69_0.tar.bz2
+
+- name: springlobby
+  buildsystem: cmake-ninja
+  sources:
+  - type: git
+    url: https://github.com/springlobby/springlobby.git
+    tag: '0.270'
+    commit: '48257e6890e360e5d947a3fee68d0d48b2ca1846'
+  - type: file
+      path: springlobby.appdata.xml
```

Comment:
This needs to be on `type` indentation level.

Reply:
Sorry, forgot to fixup that commit